components/browser/search/src/main/java/mozilla/components/browser/search/provider/localization/LocaleSearchLocalizationProvider.kt

```diff
@@ -11,9 +11,9 @@ import java.util.Locale
  * default languageTag.
  */
 class LocaleSearchLocalizationProvider : SearchLocalizationProvider() {
-    override val language: String = Locale.getDefault().language
+    override var language: String = Locale.getDefault().language
 
-    override val country: String = Locale.getDefault().country
+    override var country: String = Locale.getDefault().country
```

Comment:
I think an easier way to fix the issue would be to avoid using a backing field for this property (which will be set only once) and instead use a getter (that will be evaluated on every read) here:

```
override var language: String
    get() = Locale.getDefault().language

override val country: String
    get() = Locale.getDefault().country
```

Reply:
Will make the change